extension/js/background.js

```diff
@@ -176,62 +224,87 @@ function handleMessage(data, sender, sendRespones) {
         url: data.url,
     };
     //console.log(item)
-    db.put(item, function callback(err, result){
-        if(!err) {
+    db.put(item, function callback(err, result) {
+        if (!err) {
             console.log('Successfully stored the page: ' + data.url)
-        }
-        else {
+        } else {
             console.log('Error on URL: ' + data.url)
         }
 
     });
-}*/
+}
 
-/*function show_url(){
+function show_url() {
     db.allDocs({
-       include_docs: true, descending:true 
-    }, function(err,doc){
+        include_docs: true,
+        descending: true
+    }, function (err, doc) {
         console.log(doc);
     });
-}*/
+}
 
-/*function search_pouch(query, text, cb, suggestCb ){
-    //console.log(query)
+function search_pouch(query, text, cb, suggestCb) {
+    console.log(query);
+    // ,
+        // filter: function (doc) {
```

Comment:
if this comment will be useful can we express why or set an expiration date to it?